mucommander-viewer-text/src/main/java/com/mucommander/viewer/text/TextViewer.java

```diff
@@ -106,6 +105,7 @@ public void setFrame(final FileFrame frame) {
 
         getInputMap(JComponent.WHEN_IN_FOCUSED_WINDOW).put(KeyStroke.getKeyStroke(KeyEvent.VK_M, ActionEvent.CTRL_MASK), CUSTOM_FULL_SCREEN_EVENT);
     	getActionMap().put(CUSTOM_FULL_SCREEN_EVENT, new AbstractAction() {
+                @Override
```

Comment:
indentation?

Reply:
I autoformated class by mistake, but I hope it's ok. Btw. I'm resolving conversations, not sure if that is how to mark requested change was done. (This is actually my second PR on GitHub)

Reply:
formatting looks good now, thanks. as for resolving conversations, what I usually do is replying Done like on Gerrit and let the commenter resolve the conversation - I'm not sure if there's a "right way" or a standard way for this though.